### DISCUSSION TITLE
breeze css and normal blade views meess up when i add    presets: [preset], in main tailwind config.

### DISCUSSION THREAD

**Discussion by CushtyDev** [filamentphp/filament#13832] [Help]
### Package

Other

### Package Version

3.2.0

### How can we help you?

hi i have done some more fiddling around and if I add the preset on the default tailwind config the css messes up on the Breeze pages and any other page using the default app.blade layout.

here is the code for the default tailwind config and I have uploaded an image of what the dashboard looks like for Breeze, as soon as i remove the preset the dash looks fine, but of course the filament table on the frontend full page livewire component loses the colors etc.

Thanks

```
import defaultTheme from 'tailwindcss/defaultTheme';
import forms from '@tailwindcss/forms';
import preset from './vendor/filament/support/tailwind.config.preset'
import typography from '@tailwindcss/typography';


/** @type {import('tailwindcss').Config} */




export default {
    presets: [preset],

    content: [

        './vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php',
        './storage/framework/views/*.php',
        './resources/views/**/*.blade.php',
        './resources/views/components/*.blade.php',
        './app/Filament/**/*.php',
        './resources/views/filament/**/*.blade.php',
        './vendor/filament/**/*.blade.php',


    ],
    darkMode: 'class', // Enable dark mode
    theme: {
        extend: {
            fontFamily: {
                sans: ['Figtree', ...defaultTheme.fontFamily.sans],
            },

        },
    },

    plugins: [forms, typography],

};
```
![Screenshot_2024-08-04_at_08 58 38 (1)](https://github.com/user-attachments/assets/258c929c-bb9e-4b9a-afb1-ea3aee0c5452)

**Comment by codemarketer** [comment]
Did you solve this? I'm having the same problem

**Reply by CushtyDev** [reply]
Nope same issue, did you manage to fix?

**Comment by sira87** [comment]
Any news on this issue? I'm having the same problem.
